docs/SetupAzureDatabase-1.md

```diff
@@ -0,0 +1,58 @@
+# Set up a database
+
+This guide explains how to set up a database for Scalar DL in Azure and how to configure helm charts for Scalar DL deployment.
```

Comment:
```suggestion
This guide explains how to set up a database for Scalar DL in Azure AKS and how to configure helm charts for Scalar DL deployment.
```

Reply:
Thank you for the suggestion, fixed in 051af67